metadata-ingestion/src/datahub/emitter/composite_emitter.py

```diff
@@ -0,0 +1,34 @@
+from typing import Callable, Optional, Sequence, Union
+
+from datahub.emitter.generic_emitter import Emitter
+from datahub.emitter.mcp import MetadataChangeProposalWrapper
+from datahub.metadata.com.linkedin.pegasus2avro.mxe import (
+    MetadataChangeEvent,
+    MetadataChangeProposal,
+)
+
+
+class CompositeEmitter(Emitter):
+    def __init__(self, emitters: Sequence[Emitter]) -> None:
+        self.emitters = emitters
+
+    def emit(
+        self,
+        item: Union[
+            MetadataChangeEvent,
+            MetadataChangeProposal,
+            MetadataChangeProposalWrapper,
+        ],
+        callback: Optional[Callable[[Exception, str], None]] = None,
+    ) -> None:
+        callback_called = False
+        for emitter in self.emitters:
+            if not callback_called:
+                emitter.emit(item, callback)
+                callback_called = True
+            else:
+                emitter.emit(item)
```

Comment:
might also make sense to add a comment explaining that we want the callback to only be called once, and we tie that call to the first emitter. so we should explicitly clarify that errors from other emitters will be suppressed 

```suggestion
        for i, emitter in enumerate(self.emitters):
            emitter.emit(item, callback if i == 0 else None)
```